### DISCUSSION TITLE
Display to PictureBox after loading

### DISCUSSION THREAD

**Discussion by jueqing1015** [SixLabors/ImageSharp#1301] [Q&A]
Hello, I am using this control for the first time. How can I display the image to the PictureBox after loading the image?

**Comment by antonfirsov** [comment]
ImageSharp is not a control :) It's a cross-platform image manipulation library. I guess you are using WindowsForms. Currently we don't provide any interop utilities with GUI libraries out of the box.

Can you provide more details about your scenario?

**Reply by jueqing1015** [reply]
I have solved this problem by other methods. I would like to ask whether the library supports the removal of round holes, lines, black edges and skew
![ENG001](https://user-images.githubusercontent.com/65677359/89595212-8feb4800-d886-11ea-8c06-4b482cd245a1.png)
